src/fabric/test/fabric2_db_security_tests.erl

```diff
@@ -63,123 +61,125 @@ setup() ->
         ]}}
     ]},
     ok = fabric2_db:set_security(Db1, SecProps),
-    {ok, Db2} = fabric2_db:open(DbName, [?ADMIN_CTX]),
-    {ok, PubDb} = fabric2_db:create(PubDbName, []),
-    {Db2, PubDb, Ctx}.
+    {ok, _} = fabric2_db:create(PubDbName, []),
```

Comment:
Seems odd that this doesn't have ?ADMIN_CTX as an option when we pass it for the normal database.

Reply:
Good catch. Will add it.

Reply:
I think technically it wouldn't matter since we reopen the db using the name later on with various `#user_ctx` combinations.

In the regular db case we relied on the Db handle returned from create to have the ?ADMIN_CTX set to be able to set security on it without having to use an extra `open/2` call.